### DISCUSSION TITLE
Serial issue for Pico with last BIPES version under Chromebook ?

### DISCUSSION THREAD

**Discussion by jibeer** [BIPES/BIPES#108] [Q&A]
Hello you all !

After several attemps on my Chromebook, I dit not succeded to connect my Raspberry Pico with the latest online version of Bipes. (no serial port selection  window poping up)
Old version of Bipes (the one mentionned in the boards help page) allows to connect to pico over serial with my Chromebook, but I'd like to use the  latest version of Bipes....

Any hints ?

**Comment by rafaelaroca** [comment]
Hello, @jibeer 
Could you help us with the "old"? Which do you mean by latest and old? Do you have a link for the "old"?
Thanks

**Comment by rafaelaroca** [comment]
Me again, @jibeer 
I just tested BIPES online with a Raspberry Pi Pico and worked perfectly. 

![image](https://user-images.githubusercontent.com/31751413/154473787-d78434e3-bc08-4a07-a6f3-27598b038870.png)


What URL are you using?
Remember that USB / WebSerial needs HTTPS
https://bipes.net.br/beta2/ui/
or
https://bipes.net.br/ide/

Also, we have made several updates on BIPES over the last few days. Maybe you could try to force a full refresh (Shift+F5 several times).

regards!
Rafael

**Comment by jibeer** [comment]
Hello All :)

Thx for the quick reply. I've just tested again with my *Chromebook, (so no
way to perform some SHIFT F5 !)* with latest version
https://bipes.net.br/beta2/ui/, *no way to get the pico serial connected*.
(no popup window for serial port selection) when clicking on the 'connect
button' on the main window (despite the board RaspberryPi Pico is correctly
selected in the board selection list)
But with this 'old' url, it's ok : https://bipes.net.br/beta2serial/ui/ :
window for serial port pops up with tty/ACM0 when clicking on button
'Connect(WebSerial)' button on the Console tab...and everything is ok, but
then I do not have all the latest features....

Let me know if you need additional info;

Regards

**Comment by rafaelaroca** [comment]
Hi @jibeer , could you please provide some information about your browser console? Does it show any error?
I just discovered that ChromeOS now runs on any PC: https://chromeenterprise.google/os/chromeosflex/

But please, if you could check if any error is shown on the console, that could be useful.

This will be useful for me or anyone for testing BIPES on ChromOS

regards
Rafael

**Reply by andouglasjr** [reply]
Hi, @rafaelaroca. The same here. When attempting to connect to the Pico W, I don't see the ttyACM0 port listed. I'm using Chrome Browser on Ubuntu 22.04.

regards
Andouglas
